ArchiSteamFarm/Localization/Strings.resx

```diff
@@ -476,6 +476,10 @@ StackTrace:
     <value>Points balance：{0}</value>
     <comment>{0} will be replaced by the points balance value (integer)</comment>
   </data>
+  <data name="ErrorGetBotPointsBalance" xml:space="preserve">
```

Comment:
No need for that, just use `Strings.WarningFailed` or something like that which I use in various places.